catalog.html

```diff
@@ -59,17 +69,17 @@ <h2 class="title">Найдено гостиниц: 38</h2>
             </select>
             <ul class="view-list">
               <li class="view-item">
-                <a class="view-button-tile" href="#">
+                <a class="view-button-tile view-button-selected" href="#">
```

Comment:
В соответствии с вышесказанным здесь будут такие классы:
```suggestion
                <a class="view-button view-button-tile view-button-selected" href="#">
```